examples/shopify.rb

```diff
@@ -0,0 +1,71 @@
+class Shopify
+  attr_reader :client
+
+  def initialize(account)
+    @client = account
+  end
+
+  def shop
+    ShopifyAPI::Session.temp(@client.uid, @client.access_token){ ShopifyAPI::Shop.current }
+  end
+  
+  def revoke_shop
+    ShopifyAPI::Base.clear_session
+  end
+
+  def fetch_all_orders
+    ShopifyAPI::Session.temp(@client.uid, @client.access_token) { ShopifyAPI::Order.find(:all) }
+  end
+
+  def fetch_specific_order(order_id)
+    ShopifyAPI::Session.temp(@client.uid, @client.access_token) { ShopifyAPI::Order.find(order_id) }
+  end
+
+  def create_fulfillment(order_id, tracking_numbers, tracking_links, shipping_company)
+    session = ShopifyAPI::Session.new(@client.uid, @client.access_token)
+    ShopifyAPI::Base.activate_session(session)
+    
+    fulfill = ShopifyAPI::Fulfillment.new( :params => {:order_id => order_id})
+    fulfill.tracking_numbers = tracking_numbers
+    fulfill.tracking_company = shipping_company
+    fulfill.tracking_urls = tracking_links
+    fulfill.prefix_options = { :order_id => order_id }
+    fulfill.save
+    fulfill.id.present ? fulfill.id : ''
+
+  rescue Exception => e
```

Comment:
`Exception` class should never be rescued. Instead, you could use `StandardError`. This is going to swallow even `NameError` or `NoMethodError` inside the specs, not the code.

Reply:
alright I will refactor it